storm-core/src/jvm/org/apache/storm/state/DefaultStateEncoder.java

```diff
@@ -15,27 +15,23 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.storm.redis.utils;
+package org.apache.storm.state;
 
 import com.google.common.base.Optional;
 
-import org.apache.commons.codec.binary.Base64;
-import org.apache.storm.state.DefaultStateSerializer;
-import org.apache.storm.state.Serializer;
-
 /**
- * Helper class for encoding/decoding redis key values.
+ * Helper class for encoding/decoding key values.
  */
-public class RedisEncoder<K, V> {
+public class DefaultStateEncoder<K, V> {
```

Comment:
HBase uses `byte[]` as raw type so we could use the same encoder to HBase State, too.